apps/app.go

```diff
@@ -135,7 +136,7 @@ func marathonAppNameToServiceName(name string, nameSeparator string) string {
 }
 
 func labelsToTags(labels map[string]string) []string {
-	tags := []string{}
+	tags := make([]string, 0, len(labels))
```

Comment:
No, this capacity is used to prepare a space for tags appended in for loop below. I'm not trying to optimize allocations of the caller of `labelsToTags`.

In fact most of our own applications do not use tagged port or has only one tag.